runner.py

```diff
@@ -6,7 +6,7 @@
 if len(sys.argv) > 1:
     _board = Board(sys.argv[1])
 else:
-    _board = Board('simple.karelmap')
+    _board = Board('levels\\09_newspaper.karelmap')
```

Comment:
This path doesn't work on my machine. I think we can replace it with the following and it should also work on your's.

```suggestion
    _board = Board(os.path.join('levels', '09_newspaper.karelmap'))
```